custom-example/custom.pri

```diff
@@ -42,7 +42,7 @@ DEFINES += QGC_ORG_NAME=\"\\\"qgroundcontrol.org\\\"\"
 DEFINES += QGC_ORG_DOMAIN=\"\\\"org.qgroundcontrol\\\"\"
 
 QGC_APP_NAME        = "Custom QGroundControl"
-QGC_BINARY_NAME     = "CustomQGroundControl"
```

Comment:
Why the name change here?

Reply:
To align with the `TARGET` here: https://github.com/mavlink/qgroundcontrol/blob/master/custom-example/custom.pri#L38. Otherwise I got build errors.

Reply:
Ah, ok. Then TARGET should change to match QGC_BINARY_NAME not the other way around.

Reply:
Ok: https://github.com/mavlink/qgroundcontrol/pull/8769/commits/f27dbb68faff09360ee90ee9228dc62af5d0b13e